weave-js/src/components/Panel2/ChildPanelExportReport/ReportSelection.tsx

```diff
@@ -0,0 +1,151 @@
+import * as w from '@wandb/weave/core';
+import React from 'react';
+
+import {useNodeValue} from '../../../react';
+import {Select} from '../../Form/Select';
+import {Icon} from '../../Icon';
+import {ChildPanelFullConfig} from '../ChildPanel';
+import {
+  customEntitySelectComps,
+  customReportSelectComps,
+} from './customSelectComponents';
+import {EntityOption, ReportOption, useEntityAndProject} from './utils';
+import {Button} from '../../Button';
+
+type ReportSelectionProps = {
+  config: ChildPanelFullConfig;
+  selectedEntity: EntityOption | null;
+  selectedReport: ReportOption | null;
+  setSelectedEntity: (entity: EntityOption) => void;
+  setSelectedReport: (report: ReportOption | null) => void;
+};
+
+export const ReportSelection = ({
+  config,
+  selectedEntity,
+  selectedReport,
+  setSelectedEntity,
+  setSelectedReport,
+}: ReportSelectionProps) => {
+  const {entityName} = useEntityAndProject(config);
+
+  // Get all of user's entities
+  const entitiesMetaNode = w.opMap({
+    arr: w.opUserEntities({user: w.opRootViewer({})}),
+    mapFn: w.constFunction({row: 'entity'}, ({row}) => {
+      return w.opDict({
+        name: w.opEntityName({entity: row}),
+        isTeam: w.opEntityIsTeam({entity: row}),
+      } as any);
+    }),
+  });
+  const entities = useNodeValue(entitiesMetaNode);
+
+  // Get list of reports across all entities and projects
+  const reportsNode = w.opEntityReports({
+    enity: w.opRootEntity({
+      entityName: w.constString(selectedEntity?.name ?? entityName),
+    }),
+  });
+  const reportsMetaNode = w.opMap({
+    arr: reportsNode,
+    mapFn: w.constFunction({row: 'report'}, ({row}) => {
+      return w.opDict({
+        id: w.opReportInternalId({report: row}),
+        name: w.opReportName({report: row}),
+        updatedAt: w.opReportUpdatedAt({report: row}),
+        projectName: w.opProjectName({
+          project: w.opReportProject({report: row}),
+        }),
+      } as any);
+    }),
+  });
+  const reports = useNodeValue(reportsMetaNode);
+
+  return (
+    <div className="mt-8 flex-1">
+      <label
+        htmlFor="entity"
+        className="mb-4 block font-semibold text-moon-800">
+        Entity
+      </label>
+      <Select<EntityOption, false>
+        className="mb-16"
+        aria-label="entity selector"
```

Comment:
**issue (accessibility):** don't use `aria-label` when there's a visible label on the page. you already defined `htmlFor` on the label element, just need to tie it to the input element with an id! Hopefully the Select comp is already set up to pass `id` through to the input element

Reply:
ooohh got it - changed!